tox.ini

```diff
@@ -1,18 +1,20 @@
 [tox]
 
 envlist =
-    flake8, twistedchecker-diff
+    flake8
 
     # Twisted 15.5 is the first version to support Python 3 (3.3+)
-    trial-py{27,py,34,35,36}-tw{155,166,current,trunk}
+    coverage-py{27,py,34,35,36}-tw{155,166,current,trunk}
+
+    coverage_combine
```

Comment:
OK, I didn't realize `tox` actually ran the environments in order.  Nevertheless, `detox` runs them all at the same time, so this is still a bad idea ;)